eventfiles/models.py

```diff
@@ -0,0 +1,29 @@
+import os
+from django.db import models
+from events.models import EventParticipant
+from django.core.exceptions import ValidationError
+
+
+def upload_to_function(instance, filename):
+    return os.path.join('files/%s/' % instance.participant_id.event_id.id, filename.split("/")[-1])
```

Comment:
maybe you can do it like:
```suggestion
    return path.join('files', instance.participant_id.event_id.id, path.basename("filename"))
```